mobile/test/java/io/envoyproxy/envoymobile/engine/testing/QuicTestServerTest.java

```diff
@@ -35,58 +35,7 @@ public class QuicTestServerTest {
-      + "           \"@type\": type.googleapis.com/envoy.extensions.filters.http.router.v3.Router\n"
-      + " clusters:\n"
-      + " - name: h3_remote\n"
-      + "   connect_timeout: 10s\n"
-      + "   type: STATIC\n"
-      + "   dns_lookup_family: V4_ONLY\n"
-      + "   lb_policy: ROUND_ROBIN\n"
-      + "   load_assignment:\n"
-      + "     cluster_name: h3_remote\n"
-      + "     endpoints:\n"
-      + "     - lb_endpoints:\n"
-      + "       - endpoint:\n"
-      + "           address:\n"
-      + "             socket_address: { address: 127.0.0.1, port_value: %s }\n"
-      + "   typed_extension_protocol_options:\n"
-      + "     envoy.extensions.upstreams.http.v3.HttpProtocolOptions:\n"
-      +
-      "       \"@type\": type.googleapis.com/envoy.extensions.upstreams.http.v3.HttpProtocolOptions\n"
-      + "       explicit_http_config:\n"
-      + "         http3_protocol_options: {}\n"
-      + "       common_http_protocol_options:\n"
-      + "         idle_timeout: 1s\n"
-      + "   transport_socket:\n"
-      + "     name: envoy.transport_sockets.quic\n"
-      + "     typed_config:\n"
-      + "       \"@type\": " + QUIC_UPSTREAM_TYPE + "\n"
-      + "       upstream_tls_context:\n"
-      + "         sni: www.lyft.com\n"
-      + "listener_manager:\n"
-      + "  name: envoy.listener_manager_impl.api\n"
-      + "  typed_config:\n"
-      + "    \"@type\": type.googleapis.com/envoy.config.listener.v3.ApiListenerManager\n";
+      "static_resources { listeners { name: \"base_api_listener\" address { socket_address { address: \"0.0.0.0\" port_value: 10000 } } api_listener { api_listener { [type.googleapis.com/envoy.extensions.filters.network.http_connection_manager.v3.HttpConnectionManager] { stat_prefix: \"api_hcm\" route_config { name: \"api_router\" virtual_hosts { name: \"api\" domains: \"*\" routes { match { prefix: \"/\" } route { cluster: \"h3_remote\" } } } } http_filters { name: \"envoy.router\" typed_config { [type.googleapis.com/envoy.extensions.filters.http.router.v3.Router] { } } } } } } } clusters { name: \"h3_remote\" type: STATIC connect_timeout { seconds: 10 } dns_lookup_family: V4_ONLY transport_socket { name: \"envoy.transport_sockets.quic\" typed_config { [type.googleapis.com/envoy.extensions.transport_sockets.quic.v3.QuicUpstreamTransport] { upstream_tls_context { sni: \"www.lyft.com\" } } } } load_assignment { cluster_name: \"h3_remote\" endpoints { lb_endpoints { endpoint { address { socket_address { address: \"127.0.0.1\" port_value: %s } } } } } } typed_extension_protocol_options { key: \"envoy.extensions.upstreams.http.v3.HttpProtocolOptions\" value { [type.googleapis.com/envoy.extensions.upstreams.http.v3.HttpProtocolOptions] { common_http_protocol_options { idle_timeout { seconds: 1 } } explicit_http_config { http3_protocol_options { } } } } } } } listener_manager { name: \"envoy.listener_manager_impl.api\" typed_config { [type.googleapis.com/envoy.config.listener.v3.ApiListenerManager] { } } }";
```

Comment:
this reads a lot worse, is it possible to undo the formatting change?

Reply:
yes, but not easily :-/

All of the yaml I snagged from pretty printed output. I didn't want to go through and manually update especially as this one doesn't often change.

Reply:
no worries, can keep as is